meeting_minutes/220113_SARIF_TC_50.md

```diff
@@ -0,0 +1,162 @@
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2021-12-09 Meeting #48](https://www.oasis-open.org/committees/document.php?document_id=69499&wg_abbrev=sarif)
+
+  Minutes were approved
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION David: Find out how to upload content to wikipedia (#507)
+  - ONGOING
+* ACTION Michael: Reach out to contacts at Google and Uber on the Code Insights Protocol topic
+  - ONGOING
+* ACTION Michael: Reach out to ParaSoft
+  - ONGOING
+* ACTION Michael: Will author and check in the roadmap / list of in- and out-of-scope features
+  - ONGOING
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+* James Spoor first time participation amd if he attends the next meeting will gain voting rights at the end of that meeting.
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Aditya Sharad and Nathan Baird both will gain voting rights at the end of next meeting.
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+* None
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+* None
+
+### 1.7.5 Members who have declared a leave of absence
+
+* None
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair David)
+
+* Scheduled Teleconferences (Thursdays at 08:00 PT / 16:00 UTC for 1.5 hours)
+  ```
+  January 27
+  ```
+* Proposed Teleconferences (Thursdays at 08:00 PT / 16:00 UTC for 1.5 hours)
+  ```
+  February 10
+  February 24
+  ```
+* Possible face-to-face meeting in 2022Q1
+
+# 3. Discussion
+
+## 3.1 Review recruitment effort to complete the technical committee
+
+* David: One new member participating in the meeting today: James Spoor is welcomed
+* James: shortly introduced himself
+* Michael: Acquired a thechnical writer to assist in the production of the errata.
+
+## 3.2 Review status on finalizing SARIF 2.1 errata
+
+* David: submitted the request to OASIS for the 15-day public review of the Errata on 2021-12-09.
+* Michael: There will be support to finalize the errata (technical writer)
+* David: Checklist for finalizing errata is in github issue [Requirements to finalize the errata #509](https://github.com/oasis-tcs/sarif-spec/issues/509)
+
+## 3.3 Review current state of ecosystem ongoing work
+
+* Eddy:
+  - Github capabilities for SARIF extensions for VS and VSCode
+  - Improve Codeflows for tool (CodeQL)
+  - New RTM version (RTM 6)
+* Michael: Highlighted the impact of some of these improvements 
+
+## 3.4 Any continued report/discussion on metrics
+
+* ONGOING otherwise no updates from the committee
+
+## 3.5 Review proposal for what's in- and out-of-scope - general discussion
+
+* ONGOING and tracked by Michael's action on the metrics / roadmap
+
+## 3.6 Status of Wikipedia page
+
+* ONGOING and David plans to provide until one of the two next meetings
```

Comment:
not sure I understand...

Reply:
@dmk42: I meant provide as synonym to publish,  but maybe best you revisit and state your intent more clearly than I achieved?

Reply:
Ah, got it!